cli/src/telemetry/session.rs

```diff
@@ -130,7 +137,10 @@ impl Session {
 
         debug!("Telemetry request done");
         match resp {
-            Ok(res) => Ok(res.status().is_success()),
+            Ok(res) => {
+                debug!("response status is {}, response is {:?}", res.status(), res);
```

Comment:
This is part of what originally tipped you off to the conflicting servers yeah?

Reply:
Yes, and I figured this is a good thing as a debug message. 
fwiw I didn't see `debug!` statements printed at all, not sure why, I think I didn't configure it right with how env_logger wants it. This was `eprintln!` while I was debugging.